.travis.yml

```diff
@@ -32,6 +32,7 @@ env:
   - CRATE=boards/edgebadge FEATURES="--features=unproven"
   - CRATE=boards/xiao_m0 EXAMPLES="--example=blink --example=usb_serial" FEATURES="--features=usb"
   - CRATE=boards/wio_lite_mg126 FEATURES="--features=unproven,usb"
+  - CRATE=boards/grand_central_m4 FEATURES="--features=unproven" EXAMPLES="--example=blinky_basic"
```

Comment:
This `EXAMPLES` section is for when we want to only test certain examples out of all the ones that could ever exist. This prevents future examples from being tested so I would like it removed please.

Reply:
I think after this last change this will be good to go.

Reply:
I made the requested change to .travis.yml